src/commands/logout.rs

```diff
@@ -10,11 +10,21 @@ pub struct LogoutCommand {
     /// The common command options.
     #[clap(flatten)]
     pub common: CommonOptions,
+
+    /// The URL of the registry to use.
```

Comment:
Should we call out that this is an advanced option and plain `registry` is preferred?

Reply:
How do you mean "call out"?

Reply:
Maybe I'm just confused; what is the purpose of these two different arguments?

Reply:
I was trying to support both `warg login --registry <registry-url>` (to break existing instructions but also migrate to `warg login <registry-url>`.

Been trying, but `clap` doesn't really support it best I can tell with the `derive` args

Reply:
Ah OK; maybe pick one of the two to `#[arg(hide)]` and comment the reasoning?

Reply:
Fixed